tests/LSP9Vault/LSP9Vault.behaviour.ts

```diff
@@ -538,6 +538,52 @@ export const shouldBehaveLikeLSP9 = (
         expect(result).to.equal(value);
       });
     });
+
+    describe("When sending value to setData", () => {
+      it("should revert when sending value to setData(..)", async () => {
+        let value = 100;
```

Comment:
here the amount is very small because everything is wei.
I would suggest use etherjs library here.

```suggestion
        let value = ethers.utils.parseEthers("1");
```